algorithms/src/main/java/com/aparamonov/algostructure/search/BFS.java

```diff
@@ -14,8 +14,8 @@
 
     public List<Node<T>> search(Node<T> node) {
 
-        List<Node<T>> visited = new LinkedList<>();
-        List<Node<T>> unvisited = new LinkedList<>();
+        List<Node<T>> visited = new ArrayList<>();
```

Comment:
Measured performance difference between those two variants and it seems that it is not significant if exists at all cause we use them in streams.